tests/test_voc_format.py

```diff
@@ -802,6 +814,47 @@ def categories(self):
                 partial(VocConverter.convert, label_map='source'),
                 test_dir, target_dataset=DstExtractor())
 
+    @mark_requirement(Requirements.DATUM_GENERAL_REQ)
+    def test_dataset_with_save_dataset_meta_file(self):
+        class SrcExtractor(TestExtractorBase):
+            def __iter__(self):
+                yield DatasetItem(id=1, annotations=[
+                    Bbox(2, 3, 4, 5, label=0, id=1),
+                ])
+
+            def categories(self):
+                label_map = OrderedDict()
+                label_map['label_1'] = [(1, 2, 3), [], []]
+                label_map['background'] = [(0, 0, 0), [], []] # can be not 0
+                label_map['label_2'] = [(3, 2, 1), [], []]
+                return VOC.make_voc_categories(label_map)
```

Comment:
I think, we should also cover VOC-specific fields in this test.

Reply:
Added VOC-specific fields to this test.